### PR TITLE
feat(prisma-fmt): write a test for crdb native types

### PR DESCRIPTION
**Pull request by tomhoule**
First test for prisma_fmt::native_types().

### CODE REVIEW

File: prisma-fmt/tests/native_types.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...ring"]},{"name":"Bool","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Boolean"]},{"name":"Bytes","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Bytes"]},{"name":"Date","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["DateTime"]},{"name":"Float4","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Float"]},{"name":"Float8","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Float"]},{"name":"Inet","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["String"]},{"name":"Int2","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Int"]},{"name":"Int4","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Int"]},{"name":"Int8","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["BigInt"]},{"name":"JsonB","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Json"]},{"name":"Oid","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["Int"]},{"name":"CatalogSingleChar","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["String"]},{"name":"Uuid","_number_of_args":0,"_number_of_optional_args":0,"prisma_types":["String"]}]"#
```
Comment (Jolg42) [inline prisma-fmt/tests/native_types.rs:14]: It's a very long one line, any way to make it prettier? (also helps with diffs)

**Review by Jolg42 — APPROVED** [review]
Looks good! added a comment

File: prisma-fmt/tests/native_types.rs (hunk 1 above)
Comment (tomhoule) [reply prisma-fmt/tests/native_types.rs:14]: It's almost never going to change, and we get a nice diff when the test fails (I saw it when I removed VarChar), so we could re-parse and prettify the JSON but it's not worth the extra compile time.

**Comment by tomhoule** [comment]
Thanks!

**Pull request merged**